src/lib/utils/queryUtil/index.ts

```diff
@@ -0,0 +1,83 @@
+interface PageQueryType {
+  latitude: number | null
+  longitude: number | null
+  zoom: number | null
+  places: number[] | null
```

Comment:
What would be encoded in `places`? I'm assuming that we would need to an ID to each one of our POI categories. Not a problem, just good to know to keep this in mind for me.

Reply:
Exactly, this is still dummy data. We should have IDs here!